pkg/services/authz/zanzana/store/migration/migrator.go

```diff
@@ -1,35 +1,58 @@
 package migration
 
 import (
-	"embed"
-	"errors"
 	"fmt"
 	"strings"
 
+	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/util/xorm"
 
+	"github.com/grafana/grafana/pkg/services/sqlstore"
 	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
 	"github.com/grafana/grafana/pkg/setting"
+	"github.com/openfga/openfga/pkg/storage/migrate"
 )
 
-func Run(cfg *setting.Cfg, typ, connStr string, fs embed.FS, path string) error {
-	engine, err := xorm.NewEngine(typ, connStr)
+func Run(cfg *setting.Cfg, dbType string, grafanaDBConfg *sqlstore.DatabaseConfig, logger log.Logger) error {
+	connStr := grafanaDBConfg.ConnectionString
+	// running grafana migrations
+	engine, err := xorm.NewEngine(dbType, connStr)
 	if err != nil {
 		return fmt.Errorf("failed to create db engine: %w", err)
 	}
+	defer engine.Close()
```

Comment:
In previos version it was `return engine.Close()`. The reason of return is because there might be an error. Now with `defer` error will be lost. So it should be like 
```suggestion
	err = engine.Close()
	if err != nil {
		return fmt.Errorf("failed to close db engine: %w", err)
	}
```